docs/api/officialContracts/tat.md

```
@@ -8,8 +8,8 @@ Contract Information
 
 |Environment|Address|ABI File|
 |--|--|--|
-|testnet|--|[📥](http://)|
-|mainnet|--|[📥](http://)|
+|testnet|--|[📥](#)|
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)